watson_developer_cloud/__init__.py

```diff
@@ -16,24 +17,17 @@
 from .watson_service import WatsonException
 from .watson_service import WatsonApiException
 from .watson_service import WatsonInvalidArgument
-from .alchemy_data_news_v1 import AlchemyDataNewsV1
-from .alchemy_language_v1 import AlchemyLanguageV1
-from .alchemy_vision_v1 import AlchemyVisionV1
```

Comment:
Instead of completely removing these, we should use deprecation messages like we do the `node-sdk` so that users who try to instantiate those classes get a deprecation message instead of just an error.

Reply:
For this we will have to add the class files back and add the deprecation warnings in the class files